hospitals.Rmd

```diff
@@ -0,0 +1,116 @@
+---
+title: "Hospitals"
+author: "Marcela"
+date: "2/25/2021"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo=FALSE, warning=FALSE)
+```
+
+```{r load_libraries}
+library(dplyr)
+library(leaflet)
+library(sf)
+library(htmlwidgets)
+library(rgdal)
+```
+
+```{r read_data}
+hospitals_bogota <- readRDS(file = "hospitals_bogota.rds")
+```
+
+```{r data_wrangling}
+hospitals_bogota <- hospitals_bogota %>% 
+  select(-c(CITY, pais, SurgRms, IntensRms, EmergRms, CTRYISOA3, ADM1ISOC, ADM1ISON, SubRegion, PRIORITY, Map)) %>% 
+  mutate(lat = as.numeric(Latitude), long = as.numeric(Longitude)) %>% 
```

Comment:
override to reduce select? eg. Latitude = as.numeric(Latitude))?

Reply:
You are right! I tried to simplify code here